.github/workflows/build.yml

```diff
@@ -135,12 +135,12 @@ jobs:
 
       - name: Generate artifact attestation
         uses: actions/attest-build-provenance@v1
-        if: ${{ inputs.release && !contains(matrix.target, 'aarch64') && contains(matrix.features, 'streams') }}
+        if: ${{ inputs.release && !contains(matrix.target, 'aarch64') && matrix.features != 'debugmozjs' }}
```

Comment:
I guess in the next step we could just rename the artifact to have `-${{ matrix.features}}` suffix, so we can upload multiple versions.

Reply:
Indeed, see https://github.com/servo/mozjs/issues/541